src/code_f2470_len_27f0.c

```diff
@@ -17,6 +33,20 @@ INCLUDE_ASM(code_f2470_len_27f0, SetNpcScale);
 INCLUDE_ASM(code_f2470_len_27f0, SetNpcCollisionSize);
 
 INCLUDE_ASM(code_f2470_len_27f0, SetNpcSpeed);
+// TODO: Fix issue with BNEZL vs BNEZ
+/*
+s32 SetNpcSpeed(script_context* script, s32 initialCall) {
```

Comment:
totally out of scope, but, given this function returns an s32 and thus should something in all cases, what should it return at the end? Also, the branch issue is probably ``if(!npcPtr) return whatever;`` update/mess with it only if you feel like.

Reply:
It should return `0` (tells si to call again next frame).

Reply:
I'm pretty sure he did, but yeah something is awry

Reply:
I gave it a shot but it's still not matchin'